src/main/java/com/pgs/client/service/AuthenticationClient.java

```diff
@@ -41,4 +47,10 @@ public Token getToken() {
                 request,
                 Token.class);
     }
+
+    private void setUpRequestBody(MultiValueMap<String, String> requestBody, String grantType) {
```

Comment:
the method always takes an empty map as a parameter; it's simpler to create it inside this method

Reply:
missed comment!